DGTCentaurMods/opt/DGTCentaurMods/board/board.py

```diff
@@ -51,7 +51,7 @@
 
 # Various setup
 try:
-    ser = serial.Serial("/dev/ttyS0", baudrate=1000000, timeout=0.2)
+    ser = serial.Serial("/dev/serial0", baudrate=1000000, timeout=0.2)
```

Comment:
If this is not a problem with the Raspberry Pi device and is only to accommodate other possible SBC devices that were not completely tested yet, can we leave it ttyS0?

Reply:
We had a discussion about that a few months ago, and this line was merged in a commit 308b782 anyway.